From: Al Viro <viro@ftp.linux.org.uk>
Date: Sat, 13 Oct 2007 21:29:47 +0000 (+0100)
Subject: Fix ohci-ssb with !CONFIG_PM
X-Git-Tag: linux-2.6.24-rc1~57^2~38
X-Git-Url: http://git.kernel.org/?p=linux%2Fkernel%2Fgit%2Fralf%2Flinux.git;a=commitdiff_plain;h=4735b37cf434175c2b7b36b3b68f1e60e8ec8527;hp=d773b33972a663cfaf066e966f87922a74088a1e

Fix ohci-ssb with !CONFIG_PM

ohci_bus_{suspend,resume} exists only if we have CONFIG_PM; do the same
thing as other subdrivers...

Signed-off-by: Al Viro <viro@zeniv.linux.org.uk>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
---

diff --git a/drivers/usb/host/ohci-ssb.c b/drivers/usb/host/ohci-ssb.c
index bc3e785..fe70e72 100644
--- a/drivers/usb/host/ohci-ssb.c
+++ b/drivers/usb/host/ohci-ssb.c
@@ -117,8 +117,10 @@ static const struct hc_driver ssb_ohci_hc_driver = {
 	.hub_status_data	= ohci_hub_status_data,
 	.hub_control		= ohci_hub_control,
 	.hub_irq_enable		= ohci_rhsc_enable,
+#ifdef	CONFIG_PM
 	.bus_suspend		= ohci_bus_suspend,
 	.bus_resume		= ohci_bus_resume,
+#endif
 
 	.start_port_reset	= ohci_start_port_reset,
 };
